corehq/tabs/tabclasses.py

```diff
@@ -1974,6 +1979,22 @@ def _get_feature_flag_items(domain):
     return feature_flag_items
 
 
+def _get_release_management_items(user, domain):
+    release_management_items = []
+
+    if user.is_domain_admin(domain) and domain_has_privilege(domain, LINKED_PROJECTS):
+        release_management_items.append({
```

Comment:
These two items are on the border of deserving a helper function rather than being repeated here and above.

Reply:
Agreed because I'm bound to use this elsewhere. I'll make that change.